labs/lab-angularjs/form/form.html

```diff
@@ -0,0 +1,10 @@
+
+
+<div class="todoform">
+  <h1>ToDo List</h1>
+  <div class="list">
+    <item ng-repeat="item in $ctrl.items" item="item" class="item"></item>
```

Comment:
Because you define `this.items` as a *function* in your controller, you need to use parens to call it, here:`item in $ctrl.items()`.